Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -786,6 +677,9 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
           vCollRefsMFT[trackIndex.getIndex()] = collisionID;
         } else if (src == GIndex::Source::ITSTPC) {
           vCollRefsTPCITS[trackIndex.getIndex()] = collisionID;
+          auto& track = tracksITSTPC[trackIndex.getIndex()];
+          vToFillITS[track.getRefITS()] = false;
+          vToFillTPC[track.getRefTPC()] = false;
```

Comment:
The vertex to tracks references table already accounts for the redundancy, i.e. the ITS or TPC tracks referred by `track.getRefITS()` and `track.getRefTPC()` will never appear in this table.